internal/provisioner/model/names.go

```diff
@@ -82,17 +82,33 @@ func (c *Contour) EnvoyRBACNames() RBACNames {
 // ComponentLabels returns labels for a Contour component.
 func (c *Contour) ComponentLabels() map[string]string {
 	labels := map[string]string{
-		"app.kubernetes.io/name":       "contour",
 		"app.kubernetes.io/instance":   c.Name,
+		"app.kubernetes.io/name":       "contour",
 		"app.kubernetes.io/component":  "ingress-controller",
 		"app.kubernetes.io/managed-by": "contour-gateway-provisioner",
 	}
 
+	for k, v := range CommonLabels(c) {
+		labels[k] = v
+	}
```

Comment:
Agree with @sunjayBhatia's previous comment that we probably don't want users to be able to overwrite the above labels, so I'd start with the user-provided labels, then add the app.kubernetes.io ones.

Reply:
done